packages/core/src/modules/dids/methods/peer/peerDidNumAlgo1.ts

```diff
@@ -4,7 +4,7 @@ export function didDocumentJsonToNumAlgo1Did(didDocumentJson: Record<string, unk
   // We need to remove the id property before hashing
   const didDocumentBuffer = JsonEncoder.toBuffer({ ...didDocumentJson, id: undefined })
 
-  const didIdentifier = MultiBaseEncoder.encode(MultiHashEncoder.encode(didDocumentBuffer, 'sha-256'), 'base58btc')
+  const didIdentifier = MultiBaseEncoder.encode(MultiHashEncoder.encode(didDocumentBuffer, 'SHA-256'), 'base58btc')
```

Comment:
Yes, if the multibase encoder is exported (which I am fairly sure it is) this will break on types. 

Do we want to keep it as `sha-256` or support `SHA-256` | `sha-256` | `sha256`?

Reply:
Let's for now do backwards compat (so also lowercase sha-256). For sha-1 we only have to support the upper case variant. In 0.6 we can remove the lower case variant

Reply:
Kept it as `sha-256` and when checking it, I do `hashName.toUpperCase() === 'SHA-256'`